<ui version="4.0" >
 <class>winkladrClass</class>
 <widget class="QMainWindow" name="winkladrClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" name="gridLayout" >
    <item row="0" column="0" >
     <widget class="QTableView" name="tableView" />
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>Файл</string>
    </property>
    <addaction name="actionLoad_form_DBF" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuTest" >
    <property name="title" >
     <string>Проверка</string>
    </property>
    <addaction name="actionInputAddress" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuTest" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <action name="actionExit" >
   <property name="text" >
    <string>Выход</string>
   </property>
  </action>
  <action name="actionLoad_form_DBF" >
   <property name="text" >
    <string>Загрузить из  DBF...</string>
   </property>
  </action>
  <action name="actionInputAddress" >
   <property name="text" >
    <string>Ввод адреса</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
